modules/installation-vsphere-machines.adoc

```diff
@@ -118,7 +118,7 @@ endif::openshift-origin[]
 .. Right-click the name of your datacenter.
 .. Click *New Folder* -> *New VM and Template Folder*.
 .. In the window that is displayed, enter the folder name. The folder name must
-match the cluster name that you specified in the `install-config.yaml` file.
+match the folder name that you specified in the `install-config.yaml` file or, if you did not specify a folder name, the infrastructure ID.
```

Comment:
Not sure if this is acceptable for UPI but it is probably simpler to say "if you didn't specify an existing folder in the install-config, then create a folder..."

```suggestion
the infrastructure ID.
```

One reason I say this is because the instructions technically describe how to create a top-level folder where users could specify a nested folder